### PR TITLE
Small UI Fixes

### PR DESCRIPTION
**Pull request by thuynganvu**

### CODE REVIEW

File: app/controllers/RecruiterController.php [hunk 1]
```diff
@@ -119,7 +119,8 @@ public static function home() {
         'company' => $companyprof,
         'stats' => $stats,
         'messages' => toJSON($messages),
-        'jobs' => $jobListings
+        'jobs' => $jobListings,
+        'id' => $myId
```
Comment (coollog) [inline app/controllers/RecruiterController.php]: We should follow the convention that if the id is a MongoId, we have it be called '_id' (with the underscore).

**Comment by coollog** [comment]
Should put in the pull request description the issue tag.

#272

**Comment by thuynganvu** [comment]
Just edited.

**Comment by coollog** [comment]
LGTM merged

**Pull request merged**